server_admin/topics/groups.adoc

```diff
@@ -1,38 +1,50 @@
+
 == Groups
 
-Groups in {project_name} allow you to manage a common set of attributes and role mappings for a set of users.
-Users can be members of zero or more groups.
-Users inherit the attributes and role mappings assigned to each group.  To manage groups go to the `Groups` left menu
-item.
+Groups in {project_name} manage a common set of attributes and role mappings for each user. Users can be members of any number of groups and inherit the attributes and role mappings assigned to each group.
+
+To manage groups, click the `Groups` left menu item.
```

Comment:
We will need to agree on the wording for this type of thing.  How about "Click `Groups` in the left menu."